planetiler-core/src/test/java/com/onthegomap/planetiler/PlanetilerTests.java

```diff
@@ -1663,6 +1663,7 @@ public void processFeature(SourceFeature source, FeatureCollector features) {
       .addOsmSource("osm", tempOsm)
       .addNaturalEarthSource("ne", TestUtils.pathToResource("natural_earth_vector.sqlite"))
       .addShapefileSource("shapefile", TestUtils.pathToResource("shapefile.zip"))
+      .addGeoPackageSource("geopackage", TestUtils.pathToResource("geopackage.gpkg"), null)
```

Comment:
Could we add something to verify these features actually make it to the output, for example something in the mock profile above that emits some features for the stations and assert they show up in output below? For each new user-facing feature want to make sure there's one integration test here that verifies it works end to end.